content/docs/rfcs/11/README.md

```diff
@@ -9,45 +9,63 @@ contributors:
   - Sanaz Taheri <sanaz@status.im>
 ---
 
-`WakuRelay` is part of the gossip domain for Waku. It is a thin layer on top of GossipSub.
+<!-- A note to contributors:
+Please make use of the following best practices when adding to this document:
+- semantic line breaks as per https://sembr.org/
+- RFC key words as per RFC 2119 (https://www.ietf.org/rfc/rfc2119.txt) -->
 
-**Protocol identifier***: `/vac/waku/relay/2.0.0-beta2`
+`WakuRelay` specifies a [Publish/Subscribe approach](https://docs.libp2p.io/concepts/publish-subscribe/) to peer-to-peer messaging with a strong focus on privacy, censorship-resistance, security and scalability.
+Its current implementation is a minor extension of the [libp2p GossipSub protocol](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/README.md) and prescribes gossip-based dissemination.
+As such the scope is limited to defining a separate [`protocol id`](https://github.com/libp2p/specs/blob/master/connections/README.md#protocol-negotiationn) for `WakuRelay`, establishing privacy and security requirements, and defining how the underlying GossipSub is to be interpreted and implemented within the Waku and cryptoeconomic domain.
+
+**Protocol identifier**: `/vac/waku/relay/2.0.0-beta2`
 
 # Security Requirements
 
-The `WakuRelay` protocol is designed to provide the following security properties under a static [Adversarial Model](#adversarial-model). Note that data confidentiality, integrity, and authenticity are currently considered out of the scope of the `WakuRelay` protocol and must be handled by the upper-level protocols such as [WakuMessage](https://github.com/vacp2p/specs/blob/master/specs/waku/v2/waku-message.md). 
+The `WakuRelay` protocol is designed to provide the following security properties under a static [Adversarial Model](#adversarial-model).
```

Comment:
Formatting - the spacing looks bigger than one newline but maybe this is just GH preview? (Same in other places, but doesn't look like newline or newspace)

Reply:
Mmm, interesting. There's only a single LF-character each after lines 24 and 25. Think this may be a preview thing - there were double LFs elsewhere, which I've fixed in aeae2b5.